hourly_watch.py

```diff
@@ -15,6 +15,50 @@
 
 
 # Ska-specific watchers
+class SkaURLWatch(JobWatch):
+    def __init__(self, task, maxage_hours, url=None,):
+        self.type = 'URL'
+        self.basename = url
+        super(SkaURLWatch, self).__init__(task, url, maxage=maxage_hours * HOURS)
+
+    @property
+    def headers(self):
+        if not hasattr(self, '_headers'):
+            try:
+                import urllib2
+                response = urllib2.urlopen(self.basename)
+                self._headers = response.headers
+            except:
+                self._exists = False
+                self._headers = None
+        return self._headers
+
+    @property
+    def exists(self):
+        if not hasattr(self, '_exists'):
+            self._exists = self.headers is not None
+        return self._exists
+
+    @property
+    def age(self):
+        if not hasattr(self, '_age'):
+            if self.headers is not None:
+                import time
+                from Chandra.Time import DateTime
+                if 'Last-Modified' in self.headers:
+                    self.filetime = time.mktime(self.headers.getdate('Last-Modified'))
+                else:
+                    self.filetime = time.mktime(self.headers.getdate('Date'))
```

Comment:
> For content that has 'Last-Modified' do you then want a separate class? I think there is value added in that attribute for content that has it.

That sounds like a good idea and should simplify things.  You have one class that is just asking whether the URL loads ok (i.e. is the server running?), and another which is looking for the `modified-date` of content, where you know from testing that it should have that header keyword.  In this latter case I guess you can still be paranoid and verify that `last-modified` is available.  If not then just set `self._age` to something big like 100 days.  That will key us to check why that content no longer has `last-modified`.  (Right now jobwatch will choke if `age` is not a number, and I don't think we need to change that).